.circleci/config.yml

```diff
@@ -223,7 +223,7 @@ jobs:
             make docker.all generate_yaml
       - run: bin/testEnvRootMinikube.sh wait
       - run: docker images
-      - run: make test/minikube/noauth/e2e_pilot HUB="${HUB}" TAG="${TAG}"
+      - run: make test/minikube/auth/e2e_pilot_alpha1 HUB="${HUB}" TAG="${TAG}"
```

Comment:
Why change it to auth ? That's still pretty flaky. Please keep the auth separated.

Reply:
There's no script to run v1alpha1 without auth